includes/footer_BACKUP_11332.php

```diff
@@ -0,0 +1,152 @@
+<div class="body-container footer" style="" xmlns:background-image="http://www.w3.org/1999/xhtml">
```

Comment:
I can't seem to find where Alex's comment went, but like he said, it looks like some intermediary merge files were left over.

<hr>

> blueal requested changes on this pull request.
> 
> There seems to still be some git conflicts/files leftover